tests/framework/utils/k8s_helper.go

```diff
@@ -86,7 +94,23 @@ func CreateK8sHelper(t func() *testing.T) (*K8sHelper, error) {
 	return h, err
 }
 
-var k8slogger = capnslog.NewPackageLogger("github.com/rook/rook", "utils")
+var (
+	k8slogger  = capnslog.NewPackageLogger("github.com/rook/rook", "utils")
+	kubectlCmd = GetKubectlCmd()
+	// RetryLoop params for tests.
+	RetryLoop = getRetryLoopNumber()
+	// RookTestOpenshift variable which define testing on openshift
+	RookTestOpenshift = os.Getenv("ROOK_TEST_OPENSHIFT") == "true"
+)
+
+func getRetryLoopNumber() int {
+	// For openshift execution we need to increase retry loop as deployment
+	// takes more time
+	if RookTestOpenshift {
+		return 60
```

Comment:
Instead of a single openshift environment variable, how about a different env variable for settings like this? The retry loop doesn't depend on openshift, but really depends on the speed of the cluster.